### DISCUSSION TITLE
支持拖拽添加皮肤

### DISCUSSION THREAD

**Discussion by XYZ6656** [Meloong-Git/PCL#362] [提问] [answered]
支持拖拽添加皮肤,可以保存之前用过的皮肤进行切换

**Comment by WForst-Breeze** [comment]
个人意见：
前者可以丢一个反馈问问龙猫打不打算(/▽＼)
后者“保存皮肤进行切换”功能龙猫并不打算制作，见[**#343**](https://github.com/Hex-Dragon/PCL2/issues/343)

**Comment by Pigeon0v0** [comment, marked as answer]
前往 Issues 列表提新功能提案
